README.md

```diff
@@ -49,16 +85,211 @@ grey_fg2 = 20% lighter than grey
+return M
 ```
+
+## Override a highlight
+
+- `M.Polish` used to override highlights since `base46` will generate them
+- to override a highlight you should specify the highlight
+
+```lua
+M.polish_hl = {
+ ["@keyword"] = { bold = true}
+ String = { italic = false }
+}
+```
+
+## Setting highlightings
+
+in `base46` you can set the highlighting in diffrent way
+
+```lua
+-- ...
+M.base_30 = {
+-- ...
+ white = "#FFFFFF",
+ yellow = "yellow",
+-- ...
+}
+M.base_16 = {
+-- ...
+ base00 = "#000000",
+ base05 = "blue"
+-- ...
+}
+-- ...
+```
+
+### Base46 table options
+
+- specify the the highlightigs options for each highilght
+- you can set these options to override highlightings only in `M.polish_hl`
+  property
+
+| highlighting    | Role                                                               | Type      | Value                                       |
+| --------------- | ------------------------------------------------------------------ | --------- | ------------------------------------------- |
+| `fg`            | foreground                                                         | `string`  | color name or hex code or "none"            |
+| `bg`            | background                                                         | `string`  | color name or hex code or "none"            |
+| `sp`            | color of underlines. (more info `:h guisp`)                        | `string`  | color name or hex code or "none"            |
+| `link`          | link a highlight to another highlight options                      | `string`  | highlight name or "none"                    |
+| `italic`        | text style                                                         | `boolean` | boolean value                               |
+| `bold`          | text style                                                         | `boolean` | boolean value                               |
+| `blend`         | combining colors to create a new color (not often used in nvchad)  | `number`  | integer between 0 and 100, level of opacity |
+| `standout`      | decorations                                                        | `boolean` | boolean value                               |
+| `underline`     | decorations                                                        | `boolean` | boolean value                               |
+| `undercurl`     | decorations                                                        | `boolean` | boolean value                               |
+| `funderdouble`  | decorations                                                        | `boolean` | boolean value                               |
+| `underdotted`   | decorations                                                        | `boolean` | boolean value                               |
+| `underdashed`   | decorations                                                        | `boolean` | boolean value                               |
```

Comment:
all decorations from `underline` to `underdash` are for `sp` only, `strikethrough` is well, strikethrough texts, reverse is to reverse the colors of everything. This can be better explained by linking to vimdocs